### ISSUE TITLE
[Feature Request] Auto Accept Rewards

### ISSUE THREAD

**Issue by Epb7304** [lorddusk/HQM#692]
TL/DR:
add a new form of trigger quest where the player will automatically be given the reward on quest completion, without them clicking the claim reward button

What:
_add a new form of trigger quest where the player will automatically be given the reward on quest completion, without them clicking the claim reward button_, this will require the removal of the _pick one_ option as the player would not be able to select which one they want. The quest would also need to be on a small cooldown if repeatable so as to not overload the client.

I would even be willing to give up giving items to the player, and just be able to modify reputation and execute commands

This quest would also likely not trigger the quest complete sound effect

Why:
This would improve the mod as it would allow for better integration with potential 'bad rewards' as currently, it is not possible to force the player to decrease their reputation since they need to click the claim reward button. It is also not possible to run commands using HQM without the player directly activating them with the claim reward button. Now why specifically would commands and reputation be useful to change without player intervention? Because it allows a potential character to act through the quests, with the reputation representing how much the character likes you or not. For example a creature native to the nether not being pleased by you killing blazes and decreasing your reputation when killing them. Or bulk-running initialization commands like scoreboards and gamerules. (yes I know that last part could just be done via a datapack, but not if I want editable settings via HQM)

Alternative:
make the current trigger quest (where it is always invisible) auto-claim rewards on complete, and since it already doesn't have sound, it works out
